sdk/search/Azure.Search.Documents/tests/Samples/Sample07_VectorSearchCommons.cs

```diff
@@ -69,7 +105,7 @@ public static Hotel[] GetHotelDocuments()
                 // Add more hotel documents here...
             };
         }
-        #endregion
+#endregion
```

Comment:
Nit: this actually should be tabbed to align with the opener, but not a big deal.